src/SeleniumLibrary/__init__.py

```diff
@@ -129,8 +133,10 @@ class SeleniumLibrary(DynamicCore):
 
     Examples:
 
-    | `Click Element` | //div[@id="container"] |
-    | `Click Element` | (//div)[2]             |
+    | `Click Element` | //div[@id="container"] | Click Div With id : container |
+    | `Click Element` | (//div)[2]             | Click the second div in the document |
+    | `Click Element` | xpath=//*[contains(text(),'ABC')] | Click element containint text ABC | 
```

Comment:
Could you make the column same size by adding spaces.  The first two examples, column two and three needs some adjustments, because last example has longer column width.